test/e2e-common.sh

```diff
@@ -43,6 +43,9 @@ INSTALL_MONITORING=0
 INSTALL_CUSTOM_YAMLS=""
 
 UNINSTALL_LIST=()
+KNATIVE_DEFAULT_NAMESPACE="knative-serving"
+# This the namespace used to install Knative Serving. Use generated UUID as namespace.
+E2E_SYSTEM_NAMESPACE=$(uuidgen | tr 'A-Z' 'a-z')
```

Comment:
I guess we don't do that much here though
```suggestion
readonly KNATIVE_DEFAULT_NAMESPACE="knative-serving"
# This the namespace used to install Knative Serving. Use generated UUID as namespace.
readonly E2E_SYSTEM_NAMESPACE=$(uuidgen | tr 'A-Z' 'a-z')
```

Reply:
I make E2E_SYSTEM_NAMESPACE configurable as a param for the script as well, so I will add readonly when setting E2E_SYSTEM_NAMESPACE with the param.